code/clement/django/lab03/django_pokedex/pokedex_app/models.py

```diff
@@ -0,0 +1,33 @@
+from django.db import models
+
+# Create your models here.
+# Create an app pokedex and add two models to store our pokemon, Pokemon and PokemonType.
+class PokemonType(models.Model):
+    name = models.CharField(max_length=300)
```

Comment:
No pokemon type has a name this long, could reduce the max_length to prevent potential abuse.